src/main/kotlin/org/rust/ide/inspections/fixes/QualifyPathFix.kt

```diff
@@ -33,4 +37,11 @@ class QualifyPathFix(
         importInfo.insertExternCrateIfNeeded(path)
         path.replace(newPath)
     }
+
+    override fun generatePreview(project: Project, previewDescriptor: ProblemDescriptor): IntentionPreviewInfo {
+        if (importInfo is ImportInfo.ExternCrateImportInfo && importInfo.needInsertExternCrateItem) {
+            return IntentionPreviewInfo.EMPTY
+        }
```

Comment:
Is it really important? We decided that it's not so important to show imports. Extern crate item looks similar